Source/SwiftLintFramework/Models/LinterCache.swift

```diff
@@ -195,13 +233,8 @@ private extension StyleViolation {
 }
 
 internal extension Dictionary where Key == String {
-    func toJSON() throws -> String {
+    func toJSON() throws -> Data {
```

Comment:
Nice, I didn't realize we were converting the JSON data to a string then back to data again without ever needing the intermediate string representation. 👌